package.json

```diff
@@ -9,6 +9,7 @@
   "license": "EPL-1.0",
   "bugs": "https://github.com/redhat-developer/vscode-xml/issues",
   "preview": true,
+  "enableProposedApi": true,
```

Comment:
we can only use it in insiders. The marketplace rejects publishing when enableProposedApi:true

Reply:
We need to wait for microsoft/vscode#105394